roles/ceph-osd/templates/ceph-osd-run.sh.j2

```diff
@@ -24,6 +24,9 @@ fi
   --rm \
   --net=host \
   --cap-add SYS_ADMIN \
+  {% if ansible_os_family == "Debian" -%}
```

Comment:
why not "Ubuntu" instead? `ansible_distribution == 'Ubuntu'`? I don't think Debian uses Apparmor since it's an Ubuntu thing.